src/UI/Component/Panel/Factory.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+/* Copyright (c) 2016 Timon Amstutz <timon.amstutz@ilub.unibe.ch> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Panel;
+
+/**
+ * This is how the factory for UI elements looks. This should provide access
+ * to all UI elements at some point.
+ */
+interface Factory {
+
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *      Standard Panels are used in the Center Content section to group content.
+	 *   composition: >
+	 *      Standard consist of a title and a content section. The structure of this content might be varying from Standard
+	 *      Panel to Standard Panel. Standard Panels may contain Sub Panels.
+	 *   rivals:
+	 *      Cards: >
+	 *        Often Cards are used in Decks to display multiple uniformly structured chunks of Data horizontally and vertically.
+	 *
+	 * rules:
+	 *   usage:
+	 *      1: In Forms Standard Panels MUST be used to group different sections into Form Parts.
+	 *      2: Standard Panels SHOULD be used in the Center Content as primary Container for grouping content of varying content.
+	 * ---
+	 * @param string $title
+	 * @param mixed $content \ILIAS\UI\Component\Component[] | \ILIAS\UI\Component\Component
```

Comment:
I think the CLASS1|CLASS2 notation could be used instead of mixed, so you could pull the part in the back to the front. You could also `use \ILIAS\UI\Component\Component` to get `Component[]|Component`.